tests/locations/test_signals.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+import json
+from unittest import mock
+
+import pytest
+from django.conf import settings
+from django.utils import timezone
+from locations import models
+from locations import signals
+
+
+@pytest.mark.django_db
+@mock.patch("locations.signals._notify_administrators")
+def test_report_failed_fixity_check(_notify_administrators: mock.Mock):
+    package = models.Package.objects.create(
+        current_location=models.Location.objects.create(
+            space=models.Space.objects.create()
+        )
+    )
+    sender = None
+    kwargs = {
+        "uuid": str(package.uuid),
+        "location": "/tmp",
```

Comment:
Is the `location` still necessary in this implementation?

Reply:
Nope as long as I am not asserting it.